src/components/search/discoveryArea.tsx

```diff
@@ -17,6 +17,7 @@ import {
   updateAll,
 } from "./helper/ParameterList";
 import FilterPanel, { grouped } from "./filterPanel/filterPanel";
+import { c } from "nuqs/dist/serializer-C_l8WgvO";
```

Comment:
See above comment

Reply:
Thanks for catching that! Those were auto-imports, and I've removed them. As for the commit related to the reset button, you can view it in [PR #301](https://github.com/healthyregions/SDOHPlace/pull/301) instead.